iocage/cli/create.py

```diff
@@ -53,21 +53,28 @@ def validate_count(ctx, param, value):
 
 @click.command(name="create", help="Create a jail.")
 @click.option("--count", "-c", callback=validate_count, default="1")
```

Comment:
`-c` will create that many jails. So if you issue `iocage create -c 20 -r 11.0-RELEASE tag="foo"` you will get 20 jails, labeled `foo_1` to `foo_20`. I'll leave the wording up to you :P